lib/tool_shed/galaxy_install/tools/tool_panel_manager.py

```diff
@@ -326,7 +327,8 @@ def handle_tool_panel_section( self, toolbox, tool_panel_section_id=None, new_to
         # tool panel config (e.g., tool_conf.xml, which may have getext).  If new_tool_panel_section_label
         # is received, a new section will be added to the tool panel.
         if new_tool_panel_section_label:
-            section_id = str( new_tool_panel_section_label.lower().replace( ' ', '_' ) )
+            new_tool_panel_section_label = escape(str( new_tool_panel_section_label ))
+            section_id = escape(str( new_tool_panel_section_label.lower()).replace( ' ', '_' ) )
```

Comment:
You are escaping `new_tool_panel_section_label` twice, you can remove `escape()` from this line.